test/extended/deployments/deployments.go

```diff
@@ -246,6 +247,29 @@ var _ = g.Describe("deploymentconfigs", func() {
 		})
 	})
 
+	g.Describe("when tagging images", func() {
+		g.AfterEach(func() {
+			failureTrap(oc, "tag-images", g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should successfully tag the deployed image [Conformance]", func() {
```

Comment:
Updated the fixture to be a test deployment, even though it doesn't have to do anything with tagging.